src/Microsoft.Health.Dicom.Web/web.config

```diff
@@ -9,7 +9,7 @@
       <requestFiltering>
         <!-- When the process is hosted in IIS, change the following limit to increase maximum content length allowed. -->
         <!-- Set to 2GB by default. -->
-        <requestLimits maxAllowedContentLength="2147483648" />
+        <requestLimits maxAllowedContentLength="4294967295" />
```

Comment:
We will not default support 4GB. We may reduce it even from 2GB to 1Gb, there maybe disk contention with file streams, the long term solution to support large files will be streaming